cmake/3rdParty/Platform/Linux/BuiltInPackages_linux_aarch64.cmake

```diff
@@ -18,7 +18,6 @@ ly_associate_package(PACKAGE_NAME xxhash-0.7.4-rev1-multiplatform
 # platform-specific:
 ly_associate_package(PACKAGE_NAME expat-2.4.2-rev2-linux-aarch64                             TARGETS expat                       PACKAGE_HASH 934a535c1492d11906789d7ddf105b1a530cf8d8fb126063ffde16c5caeb0179)
 ly_associate_package(PACKAGE_NAME assimp-5.2.5-rev1-linux-aarch64                            TARGETS assimplib                   PACKAGE_HASH 0e497e129f9868088c81891e87b778894c12486b039a5b7bd8a47267275b640f)
-ly_associate_package(PACKAGE_NAME AWSGameLiftServerSDK-3.4.2-rev1-linux-aarch64              TARGETS AWSGameLiftServerSDK        PACKAGE_HASH f4ac3f8e6e9e0fce66bb3a486d0e36078d3f3d37204a9e3e95031348a021865f)
```

Comment:
I think the Gamelift 5.0.0 was accidentally removed from the most recent update.